docker-compose.reporting.yml

```diff
@@ -0,0 +1,54 @@
+version: '2.1'
+services:
+    #
+    # mender-reporting
+    #
+    mender-reporting:
+        image: mendersoftware/reporting:mender-master
+        command: server
```

Comment:
just need `server --automigrate` here.

the basic migration will then insert the index template, so the index can be auto created when tenant data comes in.

(without it we won't get our special field mappings)